pkgs/development/libraries/volume-key/default.nix

```diff
@@ -15,9 +18,18 @@ in stdenv.mkDerivation rec {
 
   outputs = [ "out" "man" "dev" "py" ];
 
-  nativeBuildInputs = [ autoreconfHook pkgconfig gettext python3 ncurses swig ];
+  nativeBuildInputs = [ autoconf automake libtool pkgconfig gettext swig ];
 
-  buildInputs = [ glib cryptsetup nss utillinux gpgme ];
+  buildInputs = [ autoreconfHook glib cryptsetup nss utillinux gpgme ncurses ];
```

Comment:
Autoreconfhook should definitely be in nativeBuildInputs.

Reply:
placing it in `nativeBuildInputs` causes `autoreconf` to fail to find `gpgme`:
```
configure.ac:47: warning: macro 'AM_PATH_GPGME' not found in library
autoreconf: running: /nix/store/givz468l2rnq7qp42d5r471mj51sdqsd-autoconf-2.69/bin/autoconf --force
configure.ac:47: error: possibly undefined macro: AM_PATH_GPGME
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
```

Reply:
Maybe some package should be in both inputs then.